### ISSUE TITLE
[Question] Is it possible to export chats with unlimited download speeds from Premium?

### ISSUE THREAD

**Issue by giannithebest1** [telegramdesktop/tdesktop#24671]
I have purchased Telegram Premium, and the download speeds are unlimited inside of chats. But when exporting my data and downloading files inside of the export tool, the files are capped at 2-3 Mbps like before. Is there any way to utilize the unlimited download speed that Premium unlocks within the export tool? This would drastically reduce export times for many users.

**Comment by Neurotoxin001** [comment]
Are you sure that download speed is unlimited in chats? Can you test it again please
https://github.com/telegramdesktop/tdesktop/issues/24654

**Comment by giannithebest1** [comment]
I just tested with a 1215 MB file in a private chat using Telegram Desktop Version 4.0.
In a normal private chat, when downloading the file, I saw an average speed of 121.5 Mbps through the entire download, with speeds peaking at about 450 Mbps.
Using the chat export tool, I saw an average speed of 6.76 Mbps through the download of the file, with speeds peaking at 8 Mbps. During this download, there was a very obvious cap of 7.7 Mbps as seen through Windows' Task Manager.
